src/sentry/features/__init__.py

```diff
@@ -76,6 +76,7 @@
 default_manager.add("organizations:customer-domains", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:dashboards-mep", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:dashboards-rh-widget", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+default_manager.add("organizations:discord-integration", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
```

Comment:
you could probably control this using flagr `REMOTE`

Reply:
Would that be an issue when we start sending notifications?

Reply:
that doesn't happen during ingest and should be okay. we've used flagr there before